app/javascript/components/analysis/component.jsx

```diff
@@ -85,29 +88,54 @@ class AnalysisComponent extends PureComponent {
           !error &&
           location.type &&
           location.adm0 && (
-            <div className="analysis-actions">
-              {location.type === 'country' && (
+          <div className="analysis-actions">
+            {location.type === 'country' && (
+              <Button
+                className="analysis-action-btn"
+                theme="theme-button-light"
+                {...linkProps}
+                onClick={() =>
+                  track('analysisViewDashboards', {
+                    label: location.adm0
+                  })
+                }
+              >
+                  DASHBOARD
+              </Button>
+            )}
+            {activeArea ? (
+            /* TODO: if activeArea, edit in my gfw AND DONT CLEAR  ???? */
+              <div className="analysis-buttons">
                 <Button
-                  className="analysis-action-btn"
+                  className="analysis-btn dashboard"
                   theme="theme-button-light"
-                  {...linkProps}
-                  onClick={() =>
-                    track('analysisViewDashboards', {
```

Comment:
We don't want to lose this event.

Reply:
`analysisViewDashboards`? We don't lose it, it's still in the code (L98)